src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -12,12 +12,13 @@
             [puppetlabs.puppetdb.jdbc :as jdbc]
             [puppetlabs.puppetdb.query.paging :as paging]
             [puppetlabs.puppetdb.scf.hash :as hash]
+            [puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.puppetdb.scf.storage-utils :as su]
```

Comment:
Yeah I really wasn't sure what to do there, for `testutils` we also have `tutils`, `tu`, sometimes `utils`. I would prefer `sutils` in general because we use `utils` and not `u` for that namespace.